lib/api/1.1/southbound/profiles.js

```diff
@@ -31,9 +31,13 @@ function profilesRouterFactory (
      */
 
     router.get('/profiles', function (req, res) {
+        if (req.query.mac && req.query.ip) {
+            profileApiService.setNode(req.query);
+            req.query.macs = req.query.mac;
+        }
+        
```

Comment:
Should put a catch either here or in the service otherwise we'll lose any errors that occur with this call.